lib/transaction/pool.go

```diff
@@ -103,12 +107,16 @@ func (tp *Pool) AvailableTransactions(transactionLimit int) []string {
 
 	var ret []string
 	// first ouput by order older hash
-	for _, key := range tp.hashes {
+	for e := tp.hashList.Front(); e != nil; e = e.Next() {
 		if len(ret) == transactionLimit {
```

Comment:
Instead of checking `len` inside loop, how about using counter?

Reply:
@spikeekips I change to use counter instead of checking len().